index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
+    
+    </head>
+    <body>
+    <div class="mid" >
+    <h1 id="title" > 
+    freeCodeCamp Survey Form
+    
+    </h1>
+    
+    <p id="description">we need your couple of minutes to complete the survey form</p>
+    
+    <form id="survey-form" >
+    
+    <label id="name-label">Name: <input id="name" type="text"name="name" placeholder="Enter your full name" required/></label>
+
+
+      <br>
+     <label id="email-label">Email: <input id="email" type="email" name="email" placeholder="Enter your email address" required/></label>
+      <br>
+    <label id="number-label" >  <input  id="number"  type="number" min="5" max="10" placeholder=" enter a number">
+      Enter Number
+      </input>
+      </label>
+      <br>
+    <select name="dropdown" id="dropdown">
+    
+            <option value="1">Programmer</option>
+            <option value="2">NOT-A-Programmer</option>
+    
+        </select>
+        <br>
+    <input name="radio" value="1" type="radio" >Male</input>
+    
+    <input name="radio" value="2" type="radio" >Female</input>
+    
+    
+    <br><br>
+    <input  value="true"  type="checkbox">
+    Programmer
+    </input>
+    <input value="true" type="checkbox" >Developer</input>
+    <input value="true" type="checkbox" >
+    Engineer
+    </input>
+    
+    <br><br>
+    <textarea rows="4" cols=""> Enter Commments</textarea>
+    </input>
+    <br><br>
+    <button id="submit" >submit</button>
+      
+      
+      </form>
+    
+      </div>
+    
+      </body>
+    
+      </html>
```

Comment:
Format the document in a proper manner so Anyone can easily read and understand it.